cmdb/framework/models/type.py

```diff
@@ -464,12 +471,19 @@ def from_data(cls, data: dict) -> "TypeModel":
         creation_time = data.get('creation_time', None)
         if creation_time and isinstance(creation_time, str):
             creation_time = parse(creation_time, fuzzy=True)
+
+        last_edit_time = data.get('last_edit_time', None)
+        if last_edit_time and isinstance(last_edit_time, str):
```

Comment:
The fuzzy was only necessary because of the old " incorrect " data elements in the database. For "new" date fields we don't really need it. But it does not hurt either.